### DISCUSSION TITLE
Applying APP update

### DISCUSSION THREAD

**Discussion by bjames3** [community/community#54579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If a APP I purchased and am modifying comes out with a update, if I copy that APP onto the original, will Git-Hub be able to track the differences?

**Comment by GeoffreyFClark** [comment]
If you forked the app, you can sync the fork. If there are conflicts between the repositiories, you can view + resolve the merge conflicts.
